modin/core/dataframe/base/dataframe/dataframe.py

```diff
@@ -252,7 +252,7 @@ def groupby(
         passed to the groupby may be at most the number of rows in the group, and
         may be as small as a single row.
 
-        Unlike the pandas API, an intermediate “GROUP BY” object is not present in this
+        Unlike the pandas API, an intermediate `GROUP BY` object is not present in this
```

Comment:
On windows I see:
```bash
python scripts\doc_checker.py modin\core\dataframe\pandas\dataframe\dataframe.py
Output:
....
UnicodeDecodeError: 'charmap' codec can't decode byte 0x9d in position 137655: character maps to <undefined>
```

I suppose this may refer to my default encoding, but how important is it for us to have unicode characters here?

Reply:
In that case, I would suggest using double quotes as a symbol (`"`), instead of unicode-double-quotes.
Apostrophe is reserved for referencing arguments of a class or function in our doc style AFAIR.

BTW, thanks for digging that failure, it bothered me a while back (though stopped in my new OS install, no idea why).